### DISCUSSION TITLE
Data residency for multi region apps.

### DISCUSSION THREAD

**Discussion by ruchitpalrecha** [planetscale/discussion#643] [Q&A]
Hey, 

I've been using Planetscale for a while now; however, we are picking up clients that are in European regions where GDPR applies. Is there a solution that allows me to support a multi-region app where I can have users in the US and in the EU while having their respective data stored/retrieved from that location?

**Comment by mscoutermarsh** [comment]
Hi @ruchitpalrecha,

Within the same database, we currently do not have the ability to store data in different locations based on the users location. This is something you would need to implement yourself with two separate databases.

**Reply by ruchitpalrecha** [reply]
Would I just have to create 2 branches and have them instantiated with the respective connection string based on the users location?

**Reply by mscoutermarsh** [reply]
Yes! That would work.
